app/models/miq_provision_virt_workflow.rb

```diff
@@ -196,15 +196,7 @@ def validate_sysprep_field(field, values, dlg, fld, value)
     return nil unless get_value(values[field]).blank?
     "#{required_description(dlg, fld)} is required"
   end
-
-  def default_require_sysprep_enabled(_field, _values, dlg, fld, value)
-    "#{required_description(dlg, fld)} is required" if value.blank? || value == "disabled"
-  end
-
-  def default_require_sysprep_custom_spec(_field, _values, dlg, fld, value)
```

Comment:
I see. Maybe some specific notation or comment for this kind of functions would be good so everyone knows that they are used by user not us. What you think?